frontend/device/generic/device.lua

```diff
@@ -66,6 +66,10 @@ local Device = {
     canPowerOff = no,
     canAssociateFileExtensions = no,
 
+    -- Start and stop text input mode (e.g. open soft keyboard, etc)
+    startTextInput = function() return nil end,
+    stopTextInput = function() return nil end,
```

Comment:
This would be more in line with the rest of the code.
```suggestion
    startTextInput = function() end,
    stopTextInput = function() end,
```

Reply:
I see, I suppose you copied it from https://github.com/koreader/koreader/commit/25d5e9322c942862a0a6d88e6e7e06a4ddfed24e which snuck in the same principle.

Reply:
Exactly right!